dpnp/dpnp_iface_linearalgebra.py

```diff
@@ -63,46 +63,30 @@
 
 def dot(x1, x2, **kwargs):
     """
-    Dot product of two arrays. Specifically,
+    For full documentation refer to :obj:`numpy.dot`
 
-    - If both `a` and `b` are 1-D arrays, it is inner product of vectors
-      (without complex conjugation).
+    Returns the dot product of `x1` and `x2`. 
 
-    - If both `a` and `b` are 2-D arrays, it is matrix multiplication,
-      but using :obj:`dpnp.matmul` or ``a @ b`` is preferred.
-
-    - If either `a` or `b` is 0-D (scalar), it is equivalent to :obj:`dpnp.multiply`
-      and using ``numpy.multiply(a, b)`` or ``a * b`` is preferred.
-
-    - If `a` is an N-D array and `b` is a 1-D array, it is a sum product over
-      the last axis of `a` and `b`.
-
-    - If `a` is an N-D array and `b` is an M-D array (where ``M>=2``), it is a
-      sum product over the last axis of `a` and the second-to-last axis of `b`::
-
-        dot(a, b)[i,j,k,m] = sum(a[i,j,:] * b[k,:,m])
-
-    Parameters
-    ----------
-    a : array_like
-        First argument.
-    b : array_like
-        Second argument.
-    out : ndarray, optional
-        Output argument. This must have the exact kind that would be returned
-        if it was not used. In particular, it must have the right type, must be
-        C-contiguous, and its dtype must be the dtype that would be returned
-        for `dot(a,b)`. This is a performance feature. Therefore, if these
-        conditions are not met, an exception is raised, instead of attempting
-        to be flexible.
+    Limitations
+    -------
+        Parameters ``x1`` and ``x2`` is supported as :obj:`dpnp.ndarray` and the same type.
+        Arguments ``**kwargs`` is supported only with value `None`.
```

Comment:
```suggestion
        Keyword arguments ``kwargs`` are currently unsupported.
```
Please apply the change where described `kwargs`.